common/lcs/src/ser.rs

```diff
@@ -79,13 +79,28 @@ impl Serializer {
         self.output
     }
 
+    fn serialize_u32_as_uleb128(&mut self, mut value: u32) -> Result<()> {
+        use serde::ser::Serializer;
```

Comment:
is there a reason to not import this at the top?

Reply:
Yes, it would name-clash with our own Serializer.